suite-native/module-send/src/screens/SendAddressReviewScreen.tsx

```diff
@@ -7,14 +7,13 @@ import { Translation, useTranslate } from '@suite-native/intl';
 import { AccountsRootState, DeviceRootState, SendRootState } from '@suite-common/wallet-core';
 
 import {
-    selectIsFirstTransactionAddressConfirmed,
-    selectIsOutputsReviewInProgress,
+    selectIsReceiveAddressOutputConfirmed,
+    selectIsTransactionReviewInProgress,
 } from '../selectors';
 import { AddressReviewStepList } from '../components/AddressReviewStepList';
 import { SendScreen } from '../components/SendScreen';
 import { SendScreenSubHeader } from '../components/SendScreenSubHeader';
 import { SendConfirmOnDeviceImage } from '../components/SendConfirmOnDeviceImage';
-import { useShowReviewCancellationAlert } from '../hooks/useShowReviewCancellationAlert';
 
 export const SendAddressReviewScreen = ({
```

Comment:
I can get app to some weird state, when I randomly close review screens and try to do the send again


Uploading Screen Recording 2024-11-22 at 16.52.30.mov…

Reply:
There was `TrezorConnet.cancel('sign-tx')` called twice, which lead to this unconsistent behaviour. Fixed in 8839f5a337